symai/prompts.py

```diff
@@ -811,3 +811,52 @@ def __init__(self):
 """outline '- Jan. 6 select committee vote criminal referrals targeting former President Trump on Monday\n-significant step end year-plus investigation' =>The Jan. 6 select committee is reportedly planning to vote on at least three criminal referrals targeting former President Trump on Monday, a significant step from the panel as it nears the end of its year-plus investigation."""
 ])
 
+
+class SymbiaCapabilities(Prompt):
```

Comment:
You need to also adapt and test this new in-context classification feature with the SymbiaChat, since it changes its behaviour.